File: OS/2do Parcial/Project/Modeling/Project/src/com/planification/Process.java
package com.planification;

public class Process {
    private int id;
    private int time;
    private int priority;

    public Process(int id, int time, int priority) {
        this.id = id;
        this.time = time;
        this.priority = priority;
    }

    public int getId() {
        return id;
    }

    public int getTime() {
        return time;
    }

    public int getPriority() {
        return priority;
    }

    @Override
    public String toString() {
        return "Process{" +
                "id=" + id +
                ", time=" + time +
                ", priority=" + priority +
                '}';
    }
}

File: OS/2do Parcial/Planification Project/process.cpp
#include "process.hpp"

int Process::count = 1;

Process::Process(int time, int priority){
    id = count++;
    this->time = time;
    this->priority = priority;
    this->timeResponse = time;
}

Process::~Process(){

}

string Process::toString(){
    string toString = "{";
    toString += "id=";
    toString += to_string(id);
    toString += ", time=";
    toString += to_string(time);
    toString += ", priority=";
    toString += to_string(priority);
    toString += "}";
    return toString;
}